helix-term/src/ui/mod.rs

```diff
@@ -225,14 +224,6 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePi
         .build()
         .expect("failed to build excluded_types");
     walk_builder.types(excluded_types);
-    let mut files = walk_builder.build().filter_map(|entry| {
-        let entry = entry.ok()?;
-        if !entry.file_type()?.is_file() {
-            return None;
-        }
-        Some(entry.into_path())
-    });
-    log::debug!("file_picker init {:?}", Instant::now().duration_since(now));
```

Comment:
My understanding is that this log predates lazy traversal of the file tree. As it was, it would record how long it takes to build an instance of `Walk`, but, _I think_, we are more interested in how long it takes to traverse the file tree and inject all file entries into a `Picker`?